engine/src/main/resources/templates/MalwareDetection/detectedMalwareInstantEmailBody.html

```diff
@@ -0,0 +1,113 @@
+{#include MalwareDetection/insightsEmailBody}
+{#content-title}Malware Detection instant notification - {action.timestamp.toStringFormat()}{/content-title}
+{#content-body}
+<!-- Body section -->
+<tr>
+  <td class="rh-body">
+    <table class="rh-content" role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+      <tr>
+        <td>
+          <table class="rh-content" role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+            <tbody>
+
+            <!-- Number of new matched signatures section -->
+            <tr>
+              <td>
+                <table class="rh-content" role="presentation" align="center" border="0" cellpadding="0" width="100%">
+                  <tr>
+                    <td class="rh-card rh-m-danger" style="width: 10%;">
+                      <img src="https://console.redhat.com/apps/frontend-assets/email-assets/lightning-bolt-light.png" alt="" width="30">
```

Comment:
I'm not sure if the host needs to be dynamic here. It depends on whether the image is publicly accessible or not.

Reply:
This one is ok as is.